build/build-report.js

```diff
@@ -81,10 +85,15 @@ async function buildEsModulesBundle() {
     file: 'dist/report/bundle.js',
     format: 'esm',
   });
+  console.log('dist/report/bundle.js written');
 }
 
 if (require.main === module) {
-  if (process.argv[2] === '--only-standalone') {
+  if (process.argv.includes('--also-psi')) {
+    buildPsiReport();
+  }
+
+  if (process.argv.includes('--only-standalone')) {
```

Comment:
i really set us up for some shitty CLI command here lol. maybe lets do this:

1) if no args, build everything.
2) if there are args (`--psi`, `--standalone`, ...) build just that

Reply:
done